engine/jobmaster/dm/api.go

```diff
@@ -125,7 +139,7 @@ func (jm *JobMaster) QueryStatus(ctx context.Context, taskID string) *dmpkg.Quer
 // OperateTask operate task.
 func (jm *JobMaster) OperateTask(ctx context.Context, op dmpkg.OperateType, cfg *config.JobCfg, tasks []string) error {
```

Comment:
it's more safe to make this method private

Reply:
Align with QueryStatus, BinlogSchema...

Reply:
It must be combined with other functions to fullfill functionality (for example, update checkpoint for "Update" operation), if we expose it as public function, other developer may misuse it. Other functions like QueryStatus, BinlogSchema may be more self-complete.